fmt/duration_test.ts

```diff
@@ -1,64 +1,113 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 import { assertEquals, assertExists } from "../testing/asserts.ts";
-import { prettyDuration } from "./duration.ts";
+import { format, parse } from "./duration.ts";
 
 Deno.test({
-  name: "duration since epoch",
+  name: "format duration since epoch",
   fn() {
-    assertExists(prettyDuration(Date.now()));
+    assertExists(format(Date.now()));
   },
 });
 
 Deno.test({
-  name: "short duration",
+  name: "format narrow duration",
   fn() {
-    assertEquals(prettyDuration(99674), "0d 0h 1m 39s 674ms 0µs 0ns");
+    assertEquals(format(99674), "0d 0h 1m 39s 674ms 0µs 0ns");
   },
 });
 
 Deno.test({
-  name: "full duration",
+  name: "format full duration",
   fn() {
     assertEquals(
-      prettyDuration(99674, { formatType: "full" }),
+      format(99674, { style: "full" }),
       "0 days, 0 hours, 1 minutes, 39 seconds, 674 milliseconds, 0 microseconds, 0 nanoseconds",
     );
   },
 });
 
 Deno.test({
-  name: "time duration",
+  name: "format digital duration",
   fn() {
     assertEquals(
-      prettyDuration(99674, { formatType: "time" }),
+      format(99674, { style: "digital" }),
       "00:00:01:39:674:000:000",
     );
   },
 });
 
 Deno.test({
-  name: "negative duration",
+  name: "format negative duration",
   fn() {
     assertEquals(
-      prettyDuration(-99674, { formatType: "time" }),
+      format(-99674, { style: "digital" }),
       "00:00:01:39:674:000:000",
     );
   },
 });
 
 Deno.test({
-  name: "full duration ignore zero",
+  name: "format full duration ignore zero",
   fn() {
     assertEquals(
-      prettyDuration(99674, { formatType: "full", ignoreZero: true }),
+      format(99674, { style: "full", ignoreZero: true }),
       "1 minutes, 39 seconds, 674 milliseconds",
     );
   },
 });
 
 Deno.test({
-  name: "short duration ignore zero",
+  name: "format narrow duration ignore zero",
   fn() {
-    assertEquals(prettyDuration(99674, { ignoreZero: true }), "1m 39s 674ms");
+    assertEquals(format(99674, { ignoreZero: true }), "1m 39s 674ms");
+  },
+});
+
```

Comment:
Can you add some tests to verify that `parse(format(value)) === value`.

Reply:
done